### PR TITLE
Update reference lookup page

### PR DESCRIPTION
**Pull request by andrew2net**
Update reference lookup page for citing bib items in override data cache #160

### CODE REVIEW

File: author/topics/building/reference-lookup.adoc [hunk 1]
```diff
@@ -23,6 +23,8 @@ the standard document identifier needs to be encoded in a format recognized by R
 
 ISO deliverables:: `ISO(identifier)`, or any identifier prefixed with `ISO`
 
+ISO/IEC Directives:: `ISO/IEC DIR {n}`, `ISO/IEC DIR {n} + {flavor} SUP`, or `ISO/IEC DIR {flavor} SUP`. `n` - is a number of directive, `flavor` - is `ISO` or `IEC`.
```
Comment (ronaldtse) [inline author/topics/building/reference-lookup.adoc]: Can you provide the exact listing of the documents provided in the Static Cache? e.g. ISO/IEC DIR IEC SUP? Should the edition year be specified, or is it automatic? Also mention that this function depends on the Static Cache so the user won't be surprised if it is different from the version online?

**Comment by ronaldtse** [comment]
@andrew2net can you also specify how to cite these in this PR:

* https://www.iso.org/sites/directives/current/consolidated/index.xhtml
* https://www.iso.org/sites/directives/current/part2/index.xhtml
* https://www.iso.org/sites/directives/current/part1/index.xhtml

Thanks!

Ron

File: author/topics/building/reference-lookup.adoc [hunk 2]
```diff
@@ -125,6 +127,9 @@ into version control systems (e.g. Git) for faster (and more reproducible)
 continuous integration build times. It also prevents CI build failures
 caused by bibliographic sources being temporarily unavailable.
 
+=== Static cache
+
+The static cache is distributed with the `relaton` gem and always enabled. It contains some documents, like ISO/IEC Directives, which can't be fetched form websites.
```
[the remark this answers is not in the document]
Comment (andrew2net) [reply author/topics/building/reference-lookup.adoc]: @ronaldtse I don't know what user's qualification is.
The static cache distributed with this gem, so data files are stored inside this gem. To add a new document to the static cache users should have the experience to use Git and make a PR.
Do you have an image of a user's qualifications?

File: author/topics/building/reference-lookup.adoc (hunk 1 above)
Comment (andrew2net) [reply author/topics/building/reference-lookup.adoc]: > Should the edition year be specified, or is it automatic?

For now, there aren't the same documents with the same number and different editions. Do you think we could have what documents in the future?

**Comment by andrew2net** [comment]
> can you also specify how to cite these in this PR:

@ronaldtse What do you mean? I've listed all the references we have in the Static cache at this moment. Do we need anything else?

**Comment by ronaldtse** [comment]
@andrew2net it's all good now, merged. Thanks.

**Pull request merged**